geopandas/array.py

```diff
@@ -1647,7 +1647,7 @@ def _reduce(self, name, skipna=True, keepdims=False, **kwargs):
         # including the base class version here (that raises by default)
         # because this was not yet defined in pandas 0.23
         if name in ("any", "all"):
-            return getattr(to_shapely(self), name)()
+            return getattr(to_shapely(self), name)(keepdims=keepdims)
```

Comment:
BTW, while you are at it, this usage of `to_shapely` seems a bit strange (probably a left-over from the shapely/pygeos divide earlier), and is the only place in the file where we still do this. 
```suggestion
            return getattr(self._data, name)(keepdims=keepdims)
```

Reply:
And for other readers, we are essentially calling the np.ndarray any/all method here, and those have a `keepdims` argument

Reply:
makes sense, seems like we could potentially also remove `to_shapely` from array.py entirely but will leave that for another PR